frame/staking/src/migrations.rs

```diff
@@ -239,13 +317,13 @@ pub mod v9 {
 					T::VoterList::count(),
 				);
 
-				StorageVersion::<T>::put(crate::Releases::V9_0_0);
+				StorageVersion::<T>::put(ObsoleteReleases::V9_0_0);
 				T::BlockWeights::get().max_block
 			} else {
 				log!(
 					warn,
 					"InjectValidatorsIntoVoterList being executed on the wrong storage \
-				version, expected Releases::V8_0_0"
+				version, expected OldReleases::V8_0_0"
```

Comment:
What is `OldReleases` and what is `ObsoleteReleases`?

Reply:
Renamed the enum but forgot the comments 🤦